lib/shoulda/matchers/active_model/errors.rb

```diff
@@ -47,6 +47,23 @@ def record_name
           model_name.humanize.downcase
         end
       end
+
+      # @private
+      class NoRangeOrArrayDefinedForInclusionError < Shoulda::Matchers::Error
+        def self.create(attribute)
+          super(attribute: attribute)
+        end
+
+        attr_accessor :attribute
+
+        def message
+          <<-EOT.strip
+You have specified that your model's #{attribute} should/should not ensure inclusion of an array or range.
+However, the test code does not provide a proper range or array to validate against.
+Please, add `in_range` or `in_array` to your test and try again.
```

Comment:
This is a good message but I think we can do a little better. What do you think about this?

```
You are using the `validate_inclusion_of` matcher without specifying the values that the matcher should use to test the inclusion validation. You'll need to provide the matcher with either a range or an array. For instance:

    should validate_inclusion_of(:#{attribute}).in_range(0..10)
    should validate_inclusion_of(:#{attribute}).in_array(['one', 'two', 'three'])
```